js/tic_tac_toe.js

```diff
@@ -1,2 +1,168 @@
-// Your code here
 // Use the jQuery
+var App = (function() {
+
+  var $boxCell, $box1, $box2, $box3, $box4, $box5, $box6, $box7, $box8, $box9;
+  var x = 1;
+
+  var init = function() {
+
+    $boxCell = $('.box_cell');
+    $('tr').on('click', '.box_cell', App.changeColor );
```

Comment:
I think you could make one function that's triggered on the click event that does the changeColor and winner functionality. Attaching too many events to the dom makes things slow in big apps (not this one... but just in theory).